docs/example-library/meltano-basics/index.md

```diff
@@ -17,8 +17,8 @@ an explicit start_date from which you would like tap-gitlab to start importing d
 that the resulting jsonl files NOT actually include timestamps in their file names:
 
 ```shell
-meltano config tap-gitlab set start_date 2022-04-25T00:00:01Z
-meltano config target-jsonl set do_timestamp_file false
+meltano --environment=dev config tap-gitlab set start_date 2022-04-25T00:00:01Z
```

Comment:
@kgpayne I'd like this test case to be updated to include setting the config globally. I think you could drop https://github.com/meltano/meltano/blob/main/docs/example-library/meltano-basics/meltano.yml#L11 and then add the configuration to validate it does it globally. 

As it stands the integration tests aren't capturing the new behavior.

Reply:
Looks like you just added it! Approving!